src/app/shared/components/addscenario/addscenario.component.html

```diff
@@ -175,24 +178,23 @@ <h4 class="modal-title pull-left">New Scenario </h4>
                                     <i class="ion-android-arrow-dropdown"></i></div>
                                 <div *ngIf="!checkDiv('error' + (errIndex + 1))" class="showHideScen" (click)="showDiv('error' + (errIndex + 1))">Error {{errIndex + 1}}<i class="ion-android-arrow-dropright"></i></div>
                             </div>
-                            <div [id]="'error' + (errIndex + 1)" class="scenFormGroup">
-                                <!-- Error ID-->
+                            <div [formGroupName]=errIndex [id]="'error' + (errIndex + 1)" class="scenFormGroup">
+                                <!--Error ID-->
                                 <div class="form-group required">
-                                    <label class="req" [for]="errIndex">Error:</label>
-                                    <select class="form-control" [formControlName]="errIndex">
-                                        <option *ngFor="let e of errors" [ngValue]="e">{{e.name}}</option>
+                                    <label class="req" for="id">Error:</label>
+                                    <select class="form-control" formControlName="id" (ngModelChange)="fillError($event, errIndex)" (ngModel)="errors">
```

Comment:
See comment in .ts file, the `(ngModelChange)="fillError(...)"` part can be removed